### ISSUE TITLE
Wiggly lines 

### ISSUE THREAD

**Issue by guaranteeofpain** [opentoonz/opentoonz#2970]
### Issue Summary
I have same issue as  https://github.com/opentoonz/opentoonz/issues/1945

### Screenshot or Video Reference
![Снимок123](https://user-images.githubusercontent.com/59448440/71679213-16dbbd80-2d98-11ea-9f7b-77effb58dcaf.PNG)

I also work in krita, tvpaint, toonboom, photoshop and none of this software has this issue.

### System Information
* **OpenToonz Version**: 1.3
* **Operating System**: win 10 build 18362
* **CPU**: 2x intel xeon 2670 
* **Memory**: 32gb
* **Graphics Card**: nvidia geforce gtx 1070
* **Graphics Tablet**: wacon cintiq companion 2 w1310p

**Comment by artisteacher** [comment]
Do you have the same issue if you lower the accuracy value? Are lines still wavy with the other level types?

**Comment by RodneyBaker** [comment]
In your image/screen capture I see the Smoothness setting is set to .3
Can you describe the results you get of increasing that value?

Note that per @shun-iwasawa the calculations for smoothing kick in with any value over zero so you shouldn't see much more lag with say... 50... than you will with .3.

Also, if you aren't using a recent nighty to test with... the v1.3 release linked from the Opentoonz webiste was from a year ago and many improvements have been made since then in the nightly releases.

Link to nightly releases:  https://github.com/opentoonz/opentoonz_nightlies/releases

**Comment by guaranteeofpain** [comment]
> In your image/screen capture I see the Smoothness setting is set to .3
> Can you describe the results you get of increasing that value?
> 
> Note that per @shun-iwasawa the calculations for smoothing kick in with any value over zero so you shouldn't see much more lag with say... 50... than you will with .3.
> 
> Also, if you aren't using a recent nighty to test with... the v1.3 release linked from the Opentoonz webiste was from a year ago and many improvements have been made since then in the nightly releases.
> 
> Your wacom software should allow for settings unique to each application. I'm not sure what others using the same hardware will recommend however. As such you might try values of 50 etc. and see if that helps.
> 
> Link to nightly releases: https://github.com/opentoonz/opentoonz_nightlies/releases

It set to 3 due to me experementing with different parameters of brush. Smooth above 10 gives kind of good line but i didnt try to ink anything so I could be wrong. Smooth between 3 and 9 gives inconsistent result - sometimes it works sometimes it is not. But setting smooth does not resolve an issue.
Thank you for link on newer builds will try it. I also try to use another wacom tablet, maybe my cintiq is buggy.

**Comment by artisteacher** [comment]
I think there have definitely been some incompatibility issues with cintiqs - but hopefully the nightly build will be better.

**Comment by RodneyBaker** [comment]
Collecting additional information here...

Are you using multiple monitors and, if yes, in your cintiq the primary?

Concerning the responsiveness of cintiq pen (or any pen for that matter), there at least three places to consider when setting pen options; 1) The pen and pen software  2)  The operating system 3) the application.  I don't know enough about the cintiq being used to speculate much further but in the past when having issues with pen responsiveness I've almost always been able to resolve it by eliminating of the variables... reducing variation... until something worked.  I would say however that **99 times out of 100 it has been a driver that has resolved the most persistent issues.**

**Comment by guaranteeofpain** [comment]
> Collecting additional information here...
> 
> Are you using multiple monitors and, if yes, in your cintiq the primary?
> 
> Concerning the responsiveness of cintiq pen (or any pen for that matter), there at least three places to consider when setting pen options; 1) The pen and pen software 2) The operating system 3) the application. I don't know enough about the cintiq being used to speculate much further but in the past when having issues with pen responsiveness I've almost always been able to resolve it by eliminating of the variables... reducing variation... until something worked. I would say however that **99 times out of 100 it has been a driver update (or more rarely a regression to earlier driver) that has resolved the most persistent issues.**

Yes I use two displays, one of them 1080p and other is 2k cintiq. I dont want to change things in my drivers nor in my pc setup because I constantly work on projects and if something goes wrong it is loss of time and money, so maybe I'll wait for driver and use smooth 10+.
I still didnt tried to use single display and intuos, if it resolves issue I tell you.
Thanks for advice.

**Comment by gab3d** [comment]
@guaranteeofpain 
Please report back when you've tried with the latest "nightly build" of OpenToonz.

**Comment by RodneyBaker** [comment]
As #1945 is open we'll close this as a duplicate of that issue and look forward to hearing about the results.

Some of suggestions for multiple displays include making sure the cintiq is the primary monitor and (where possible) making sure both/all monitors are of the same (or at least compatible resolution.  It does makes sense that if your cintiq  is not the primary monitor there might be additional interference or delay so I'd definitely look into that.

In reporting back feel free to post here or in #1945.
We'll see it either way.

**Comment by Pilgrim1987** [comment]
I have similar issue. Wriggly lines while drawing on my cintiq 12WX on the left and perfectly straight lines (used physical ruler in both cases) while drawinf on Intuos tablet.
![image](https://user-images.githubusercontent.com/53706994/72131956-cdd6cb00-338e-11ea-98b3-a374025b2fea.png)
System Information
OpenToonz Version: 1.30
Operating System: win 10 build 17763.914
CPU: i7 8700
Memory: 32gb
Graphics Card: nvidia geforce gtx 1080
Graphics Tablet: wacom cintiq 12wx and wacom intuos 3

**Comment by RodneyBaker** [comment]
Hopefully everyone has read what resolved the issue in #1945:

> I can't be sure what happened but this issue resolved itself. I purchased a 2nd monitor and adjusted the tablet settings to just draw on one monitor and the issue is no longer present.

**Comment by Pilgrim1987** [comment]
I have no issues while using tablet, it is indeed set up to work on one monitor.
The wiggly lines appear only when I use my small cintiq 12wx. It is set up to work only on its own area/diasplay.

**Comment by Lovewyrm** [comment]
I use a cintiq 24 pro from 2018 and I have no such problems (just as a counter example).

**Comment by RodneyBaker** [comment]
@Pilgrim198
Please open a new report that specifically mentions how you are on a cintiq with one single monitor.
My concern here being that @guaranteeofpain's issue (and therefore this report) will not be resolved until such a time as drivers and settings are updated and those changes cannot be made at this time.

By opening a new report that is not a duplicate... because of the single monitor... we can address your issue directly and get it resolved.

Short of that I would say double/triple check your drivers and perhaps that will address the issue and you won't need to open another report.

**Comment by RodneyBaker** [comment]
Closing as resolved for the orginal reporter.
@guaranteeofpain Please let us know if you experience any similar problems.
